python/semantic_kernel/events/function_invoked_event_args.py

```diff
@@ -1,16 +1,45 @@
 # Copyright (c) Microsoft. All rights reserved.
 
+from typing import Optional
+
+from pydantic import Field
+
 from semantic_kernel.events.kernel_events_args import KernelEventArgs
+from semantic_kernel.functions.function_result import FunctionResult
 
 
 class FunctionInvokedEventArgs(KernelEventArgs):
```

Comment:
Does it make things more clear to have an explicit class constuctor (__init__)?

Reply:
I don't want to touch these more then I had to, there are adrs about this that we have to look at further